extern/filereaderlp/reader.cpp

```diff
@@ -892,9 +892,14 @@ void Reader::readnexttoken(bool& done) {
    done = false;
 
    if (this->linebufferpos == this->linebuffer.size()) {
-     // read next line
+     // read next line if any are left. 
      std::getline(this->file, linebuffer);
-
+     if (this->file.eof())

```

Comment:
Doesn't this now skip parsing the content of the last line, if not empty?

I was also ok with the previous solution.

Reply:
You're right; `eof` triggers on the last line, even if it has contents. I guess it's safe anyway given that the end marker appears on a line of it's own, but relying on that is iffy. Let me just revert the last commit.